### PR TITLE
cleaned up

### PR DESCRIPTION
**Pull request by Kubik42**

### CODE REVIEW

**Comment by yeekangc** [comment]
@Kubik42, I don't think you have the latest from master?

**Comment by Kubik42** [comment]
should be fine now, resolved the conflicts

File: README.adoc [hunk 1]
```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
...
+A health report will be generated automatically for all services that enable MicroProfile Health. This
+feature has already been enabled for you in the `src/main/liberty/config/server.xml` file:
+
+[source, xml, role="no_copy"]
+----
+<feature>mpHealth-1.0</feature>
+----
+
+All applications must provide an implementation of the `HealthCheck` interface, which will be used to
+verify the healthiness of the services.
```
Comment (yeekangc) [inline README.adoc]: Use 'services' instead of 'applications'?

File: README.adoc [hunk 2]
```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
...
 // =================================================================================================
 
-=== Adding a health check to the system service
+=== Adding a health check to the `system` service
+
+Before you proceed, navigate to the `start` directory where you will find the starting project:
+
+```
+cd ../start
+```
```
Comment (yeekangc) [inline README.adoc]: Is the 'cd ../start' necessary?  I can see why you removed the "Navigate back ..." sentence above now.

File: README.adoc [hunk 3]
```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
...
 
-This class implements the `HealthCheck` interface from the MicroProfile Health API. The class also implements the `call()`
-method from the interface to report the state of the service by returning a `HealthCheckResponse` object. The `system`
-service self-checks. It reads the system property of the host to report a healthy state.
-If the `wlp.server.name` system property is `defaultServer`, the state is `UP`. Otherwise, the state is `DOWN`.
+The `call()` method is used to return the health status of a particular service. In this case, you are simply
+checking if the server name is `defaultServer` and returning `UP` if it is, and `DOWN` otherwise.
+The `HealthCheckResponse.named()` method is used to indicate what service the health check is done for.
+Overal, this is a very simple implementation of the `call()` method. In a real development environment,
+you would want to orchestrate much more meaningful health checks.
```
Comment (yeekangc) [inline README.adoc]: Typo with 'Overal'.

File: README.adoc [hunk 4]
```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
...
 ----
 
-This service reports a healthy state based on two factors:
+This time, you are checking whether or not the service is in maintenance or if it's down. For simplicity,
+a custom `io_openliberty_guides_inventory_inMaintenance` MicroProfile Config property defined in the
+`resources/CustomConfigSource.json` file is used to
+indicate whether the service is in maintenance or not. This file has already been created for you.
+To check that the service is down, simply make a HTTP GET request to itself and check the status
+returned by the response. If the status is not 200, then the service is not running. Based on these
+two factors, the `isHealthy()` method returns whether or not the `inventory` service is healthy.
```
Comment (yeekangc) [inline README.adoc]: Not to itself but the `system` service?

File: README.adoc [hunk 5]
```diff
@@ -54,171 +53,122 @@ include::{common-includes}/gitclone.adoc[]
...
+- http://localhost:9080/inventory/systems
 
-Next, go to `http://localhost:9080/health` URL to see the health of the two services as well as the overall
-health outcome of the application. One check shows the state of the `system` service, and the other shows the state of
-the `inventory` service. As you might expect, both services are in the `UP` state, and the overall outcome of
-the application is `UP`.
+Visit the `http://localhost:9080/health` URL to see the health report of the two services as well as the overall
+health status of the application. One check shows the state of the `system` service, and the other shows the state of
+the `inventory` service. As you might expect, both services are in the `UP` state, and the overall health status of
+the application is `UP`:
```
[the remark this answers is not in the document]
Comment (evelinec) [reply README.adoc]: The sample response was requested by Laura. Not sure we want to remove it.

File: README.adoc (hunk 2 above)
Comment (evelinec) [reply README.adoc]: Note if this change is made, this guide will be slightly different from the other ones.

File: README.adoc [hunk 6]
```diff
@@ -15,27 +15,27 @@
 :page-related-guides: ['microprofile-config', 'microprofile-metrics', 'cdi-intro', 'rest-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
-= Adding health reports to a microservice
+= Adding health reports to microservices
 
-You'll explore how to report and check the health of a microservice with MicroProfile Health.
+Explore how to report and check the health of a microservice with MicroProfile Health.
```
Comment (evelinec) [inline README.adoc]: I thought "You'll explore" was ID edit, but now looks like we have them both in different guides. Not a big deal, thought just to mention it.

File: README.adoc [hunk 7]
```diff
@@ -54,171 +53,122 @@ include::{common-includes}/gitclone.adoc[]
...
-The `finish` directory in the root file contains the finished
-health check implementation for the services in the application. You can try out the application before
-you build your own.
+The `finish` directory in the root directory of this guide contains two services that are configured
+to use MicroProfile Health. Feel free to give them a try before you proceed.
 
-To try the application, navigate to the `finish` directory. Next, execute the following
-Maven goals to build the application and run it inside Open Liberty:
+To try out the services, navigate to the `finish` directory and then run the Maven `install` and
+`liberty:start-server` goals to build the services and run them in two Open Liberty servers:
```
[the remark this answers is not in the document]
Comment (evelinec) [reply README.adoc]: Should be 1 server for this app.

File: README.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (yeekangc) [reply README.adoc]: Do we know the why?  It should be captioned properly if we have it, which it currently isn't.

And, we should clean up the data in it with a separate/new issue too.

File: README.adoc [hunk 8]
```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
...
 ----
 
-The `org.eclipse.microprofile` package contains the required MicroProfile Health Check classes.
-
-The `@Health` annotation indicates that this class reports the state of
-`system` service through `http://localhost:9080/health` URL. The `@ApplicationScoped` annotation is
-required because the health check service must stay active within the lifetime of the application
-to report the state.
+The `@Health` annotation indicates that this particular bean implements the `HealthCheck` interface.
+By pairing this annotation with the `ApplicationScoped` context from the CDI API, the bean is discovered
```
Comment (evelinec) [inline README.adoc]: I believe this is the first time you mentioning "CDI" in this guide, you may want to spell it out.

File: README.adoc (hunk 7 above)
[the remark this answers is not in the document]
Comment (Kubik42) [reply README.adoc]: fixed

File: README.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (Kubik42) [reply README.adoc]: I dont think we need a sample response. We dont have any samples in other guides for this section. It should be clear enough from the description what the user is expected to see. I replaced the colon with a period.

File: README.adoc (hunk 1 above)
Comment (Kubik42) [reply README.adoc]: fixed

File: README.adoc (hunk 2 above)
Comment (Kubik42) [reply README.adoc]: I talked about this with Jay and Laura a while back when we discussed removing the `cd start` from the "Getting started" section. Their concern was that if the users skip the "Try what you'll build" section, then they might be lost as to where to begin. So, Jay suggested to put `cd start` in the first section of the guide that talks about implementation details.

File: README.adoc (hunk 3 above)
Comment (Kubik42) [reply README.adoc]: good catch, fixed

File: README.adoc (hunk 4 above)
Comment (Kubik42) [reply README.adoc]: oh I see now, thats kind of confusing... We are reporting whether or not the `inventory` service is healthy by checking if the `system` service is running? I'll make a note of this somewhere so its clear the prior service depends on the latter one.

File: README.adoc (hunk 6 above)
Comment (Kubik42) [reply README.adoc]: oh, the two guides I wrote had "Explore". I think this is fine

**Comment by Kubik42** [comment]
Made more edits @evelinec @yeekangc

File: README.adoc (hunk 2 above)
Comment (yeekangc) [reply README.adoc]: I seem to recall this so I am fine with it as such for Health here.  We should open up a separate issue to review this comprehensively and update as appropriate across all guides.  I will get an issue created.

File: README.adoc (hunk 3 above)
Comment (yeekangc) [reply README.adoc]: Also, I missed this -- It wouldn't be "real development environment" but "real world usage" or equivalent.

**Comment by Ahmad-Ayyoub** [comment]
@Kubik42, I wonder why u removed a lot of info :S. 

- The `system` and `inventory` words are too big in the sub-titles.
- GET -> `GET`

**Comment by yeekangc** [comment]
- The focus is not on length but adequate information and explanation for a meaningful learning experience
- GET is fine and we used that elsewhere; Can double check with Sarah if there are concerns

File: README.adoc (hunk 4 above)
Comment (yeekangc) [reply README.adoc]: Yes, that's my understanding.  Thank you.

File: README.adoc (hunk 6 above)
Comment (evelinec) [reply README.adoc]: Sure, np.

File: README.adoc (hunk 2 above)
Comment (evelinec) [reply README.adoc]: Ok, sounds good.

**Pull request merged**